src/app/explorer/explorer.component.ts

```diff
@@ -149,6 +151,8 @@ export class ExplorerComponent implements OnInit {
     let folderPath = this.selectedItem.ITEMPATH
     let jobName = this.selectedItem.ITEMNAME
 
+    this.selectedJobLink = `${this.sasjsConfig?.serverUrl}/${this.sasjsConfig?.serverType === 'SASVIYA' ? 'SASJobExecution' : 'SASStoredProcess'}/?_PROGRAM=${folderPath}/${jobName}`
```

Comment:
We are about to add one more `ServerType` to `@sasjs/utils`, so the ternary operator is not reliable